lab 2/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>My title</title>
+
+      <!-- font awesome -->
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+      <link rel="stylesheet" href="css/style.css">
+   </head>
+   <body>
+   <header>
+      <div class="collapsed-menu">
+         <div class="title menu-item">Navigation</div>
+         <div class="button menu-item"><i class="fa fa-bars" aria-hidden="true"></i></div>
+      </div>
+      <div class="menu">
+         <div class="menu-item"><a href='#'>psd websites</a></div>
+         <div class="menu-item"><a href='#'>ui kits</a></div>
+         <div class="menu-item"><a href='#'>premium psd files</a></div>
+         <div class="menu-item"><a href='#'>about</a></div>
+         <div class="menu-item"><a href='#'>blog</a></div>
+      </div>
+   </header>
+   <article>
+      <div class="page page-1">
+         <section>
+            <div class="content-main">
+               <h1>Awesome Free PSD <br> templates &amp; Resources</h1>   
+            </div>
+            <a href="#">
+               <div class="link" style="display: flex; justify-content: space-between; align-items: center;">
+                  <span>www.BlazRobar.com</span>
+                  <i class="fas fa-1x fa-angle-right"></i>
+               </div>
+            </a>
+         </section>
+      </div>
+      <div class="page page-2">
+         <div class="line first-line">
+            <div class="text">
+               <p>I'm like the Robin Hood of PSD files.</p>
```

Comment:
Better to use p.text, not necessary to wrap into div

Reply:
if so, it's neccessary to assign each p with a class. Is it really better than to wrap it into a signle div?

Reply:
Why do you need to assign each p on the page a class? You could use something like `.first-line .text { color: white }` to apply certain styles only to this inner p element with 'text' class, couldn't you?

Reply:
Yes of course. But what about styles, which are aplied only to the element with current class but not to the child ones, e.g. paddings, margins and borders?
In current situation, each p should contain padding, because in small layout this p's should be placed in two colomns. The gap between this colomns can be easily set with this padding.
it seemed to be a beautifull solution for a such problem...

Reply:
Answering your question: and what about them?) You could combine classes and selectors to apply certain styles (paddings, margins, borders) for the children of the particular element.

Actually it's hard for me to understand your thoughts. This text "I'm like the Robin Hood of PSD files" shouldn't be placed in two columns (see the provided designs). So I can't find the reasons to use this some kind of complex logic

Reply:
A lot of time had passed till a moment i understood, that i was talking about text in the last part of site (.page-2 > .third-line) while the issue is in the first-line.. Sorry about that.
Concerning the "i'm like Robin Hood..'.. I can explain that.
For me is was easier to understand this line as two boxes, sticked to each other. So i placed text and an arrow into different containers (div's), just because that it is easier for me to manage a back-color and align of them while not bothering about inner content.
Of course, if the goal is to minimize the amount of tags used, than that issue is not worth-discussing and you're right.

Reply:
The goal is to write less and gain more. You obtain understandable and clear code when you just keep it simple. So keep practicing and you will succeed) 

And of course we'll discuss all your questions on the lesson if you'd like to